examples/default/src/screens/apm/HttpScreen.tsx

```diff
@@ -0,0 +1,250 @@
+    setLoading3(true);
+    const url = 'https://httpbin.org/delete';
+    try {
+      const response = await fetch(url, {
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+
+      const responseBody = await response.json();
+      console.log('Response:', responseBody);
+
+      setLoading3(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Succeeded', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Success', 'Succeeded');
+      }
+    } catch (error) {
+      console.error('Error:', error);
+      setLoading3(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Failed', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Error', 'Failed');
+      }
+    }
+  };
+
+  const makePutCall = async () => {
+    setLoading4(true);
+    const url = 'https://httpbin.org/put';
+    const requestBody = {
+      name: 'Islam',
+    };
+    try {
+      const response = await fetch(url, {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(requestBody),
+      });
+
+      const responseBody = await response.json();
+      console.log('Response:', responseBody);
+
+      setLoading4(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Succeeded', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Success', 'Succeeded');
+      }
+    } catch (error) {
+      console.error('Error:', error);
+      setLoading4(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Failed', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Error', 'Failed');
+      }
+    }
+  };
+
+  const makePatchCall = async () => {
+    setLoading5(true);
+    const url = 'https://httpbin.org/patch';
+
+    const jsonInputString = JSON.stringify({ name: 'Islam' });
+
+    try {
+      const response = await fetch(url, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: jsonInputString,
+      });
+
+      const responseBody = await response.json();
+      console.log('Response:', responseBody);
+      setLoading5(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Succeeded', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Success', 'Succeeded');
+      }
+    } catch (error) {
+      console.error('Error:', error);
+      setLoading5(false);
+
+      if (Platform.OS === 'android') {
+        ToastAndroid.show('Failed', ToastAndroid.SHORT);
+      } else {
+        Alert.alert('Error', 'Failed');
+      }
+    }
+  };
+
+  const makeDownloadImageCall = async () => {
+    setLoading6(true);
+    const url = 'https://httpbin.org/image/jpeg';
+    try {
+      const response = await fetch(url, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'image/jpeg',
+        },
```

Comment:
I don't think we need the `Content-Type` header here since we're not actually sending any data with the request so we can safely remove it.

If you want we can change it to `Accept` to tell the server that we need the response in `image/jpeg` format, though not necessary.